src/disco/addonManager.js

```diff
@@ -36,3 +42,23 @@ export function uninstall(guid, {_mozAddonManager = window.navigator.mozAddonMan
       return;
     });
 }
+
+export function addChangeListeners(callback, mozAddonManager) {
```

Comment:
I don't think this should be plural. I could probably be convinced but from the caller's perspective this just registers a single callback that will be notified of the new status.